rules-reviewed/eap6/websphere/websphere-xml.windup.xml

```diff
@@ -145,28 +145,33 @@
                 </xmlfile>
             </when>
             <perform>
-                <iteration over="1">
-                    <classification title="WebSphere Web Application Binding" effort="0">
-                        <description>WebSphere 6 web application binding configuration file.</description>
-                        <link title="WebAppBinding javadoc" href="http://rjweb002.royalsun.com.br/configDocs/webappbnd/WebAppBinding.html"/>
-                        <tag>ejb</tag>
-                        <tag>websphere</tag>
-                        <tag>application-bindings</tag>
-                    </classification>
-                </iteration>
                 <iteration over="2">
-                    <hint title="WebSphere Web Application Bindings" effort="5">
+                    <hint title="WebSphere web application binding" effort="5">
                         <message>
                           <![CDATA[
                           This file configures the specific WebSphere Web application bindings.
 
                           Configure EAP 6 Web subsystem accordingly using the CLI interface or web console.
                           ]]>
                         </message>
-                        <link title="Configure the EAP6 Web Subsystem" href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html-single/Administration_and_Configuration_Guide/index.html#chap-Web_Subsystem"/>
+                        <link title="Configure the EAP 6 Web Subsystem" href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html-single/Administration_and_Configuration_Guide/index.html#chap-Web_Subsystem"/>
                         <link href="http://docs.jboss.org/jbossweb/7.0.x/" title="JBoss Web configuration"/>
                     </hint>
                 </iteration>
+                <iteration over="1">
+                    <when>
+                        <not>
+                            <has-hint />
```

Comment:
It's supposed to prevent adding the classification to a file where the <iteration over="2"> already put a hint.